packages/cli/src/lib/system/file-lock.ts

```diff
@@ -1,4 +1,17 @@
 import * as fs from "fs";
+import path from "path";
+
+export function createDirs(dirPath: string): void {
```

Comment:
This can be done via `fs.mkdirSync(dir, { recursive: true });`

Reply:
Nice! didn't know about that 😛